meetings/2020-12-14.md

```diff
@@ -43,10 +44,35 @@
   - 11 January 2021
 
 # Action Items (AIs)
-  - 
+  - Add SideFX CPython porting notes to the github repo
+    **Robert Vinluan**
+  - Send out a reminder to watch the github repo to the mailing list
+    **Ashley Whetter**
```

Comment:
Meta: Some folks may not be too familiar with the Github "Watch" feature, so it might me good to spell this out in the mailing list email and let folks know that they have to visit the repo, click on the "Eye" icon and enable notifications.